packages/cli-repl/src/arg-parser.spec.ts

```diff
@@ -404,6 +416,18 @@ describe('arg-parser', () => {
           });
         });
 
+        context('when providing -tlsCertificateKeyFile (single dash)', () => {
+          const argv = [ ...baseArgv, uri, '--tlsCertificateKeyFile', 'test' ];
```

Comment:
```suggestion
          const argv = [ ...baseArgv, uri, '-tlsCertificateKeyFile', 'test' ];
```
You would want to test the single dash version, right? 😅